libs/qmlglsink/CMakeLists.txt

```diff
@@ -122,43 +128,48 @@ if(QGC_ENABLE_VIDEOSTREAMING)
 				${GST_TARGET_PLUGINS}
 			)
 		elseif(LINUX)
-			set(ENV{PKG_CONFIG_PATH} "${GSTREAMER_ROOT}/lib/pkgconfig:${GSTREAMER_ROOT}/x86_64-linux-gnu/lib/pkgconfig:$ENV{PKG_CONFIG_PATH}")
+			set(ENV{PKG_CONFIG_PATH} "${GSTREAMER_ROOT}/lib/	pkgconfig:${GSTREAMER_ROOT}/x86_64-linux-gnu/lib/pkgconfig:$ENV{PKG_CONFIG_PATH}")
 			pkg_check_modules(GST IMPORTED_TARGET ${GST_TARGET_MODULES})
 		elseif(MACOS)
-			set(ENV{PKG_CONFIG_PATH} "${GSTREAMER_ROOT}/Versions/Current/lib/pkgconfig:$ENV{PKG_CONFIG_PATH}")
-			pkg_check_modules(GST IMPORTED_TARGET ${GST_TARGET_MODULES})
-		else()
-			set(ENV{PKG_CONFIG_PATH} "${GSTREAMER_ROOT}/lib/pkgconfig:$ENV{PKG_CONFIG_PATH}")
-			pkg_check_modules(GST IMPORTED_TARGET ${GST_TARGET_MODULES})
+			if (GSTREAMER_FRAMEWORK-NOTFOUND)
+				message(WARNING "Gstreamer Framework Not Found")			
+			else()
+			    set(GST_FOUND TRUE)
```

Comment:
Since the find_library uses a different mechanism to find the frame the GST_FOUND variable is not set automatically. So I set it manually here. I must be doing this wrong somehow.

Reply:
Is this specific CMakelists.txt it does see that GST_FOUND was set to TRUE further down the file.